docs/src/extend/shareable-configs.md

```diff
@@ -49,96 +60,59 @@ You should declare your dependency on ESLint in the `package.json` using the [pe
 ```json
 {
     "peerDependencies": {
-        "eslint": ">= 3"
+        "eslint": ">= 8"
```

Comment:
```suggestion
        "eslint": ">= 9"
```

It might be better to use 9 in the example as not all versions of ESLint 8 support flat config.